ipaserver/install/ldapupdate.py

```diff
@@ -63,10 +62,7 @@ def connect(ldapi=False, realm=None, fqdn=None, dm_password=None):
             fqdn, decode_attrs=False
         )
     try:
-        if dm_password:
-            conn.simple_bind(bind_dn=ipaldap.DIRMAN_DN,
-                             bind_password=dm_password)
-        elif os.getegid() == 0:
```

Comment:
Why did you remove this code?

Reply:
See comment by @rcritten in the ticket:
>Near as I can tell only schema_update() uses ldapupdate.connect() and never passes in a DM password so looks like simple bind can be dropped altogether.

Reply:
Any removed feature means that we cannot backport this PR to 4.8 branch.

Reply:
I removed the commit from PR. If requested I can submit it separately.

Reply:
Right, ipa-4-8 still uses connect with dm_password all over the place. A separate PR for "next" branch works for me.

Reply:
:+1: